tests/alcatel_aos/show_interfaces_status/alcatel_aos_show_interfaces_status_R6.yml

```diff
@@ -0,0 +1,392 @@
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: ""
+    link_trap: ""
+    port: "1/18"
+  - admin_status: ""
+    auto_negotiation: "Enable"
+    configured_duplex: "Auto"
+    configured_fec: ""
+    configured_hybrid: "NA"
+    configured_pause: ""
+    configured_speed: "Auto"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: ""
+    link_trap: ""
+    port: "1/19"
+  - admin_status: ""
+    auto_negotiation: "Enable"
+    configured_duplex: "Auto"
+    configured_fec: ""
+    configured_hybrid: "NA"
+    configured_pause: ""
+    configured_speed: "Auto"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: ""
+    link_trap: ""
+    port: "1/20"
+  - admin_status: ""
+    auto_negotiation: "Enable"
+    configured_duplex: "Auto"
+    configured_fec: ""
+    configured_hybrid: "NA"
+    configured_pause: ""
+    configured_speed: "Auto"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: ""
+    link_trap: ""
+    port: "1/21"
+  - admin_status: ""
+    auto_negotiation: "Enable"
+    configured_duplex: "Auto"
+    configured_fec: ""
+    configured_hybrid: "NA"
+    configured_pause: ""
+    configured_speed: "Auto"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: ""
+    link_trap: ""
+    port: "1/22"
+  - admin_status: ""
+    auto_negotiation: "Enable"
+    configured_duplex: "Auto"
+    configured_fec: ""
+    configured_hybrid: "NA"
+    configured_pause: ""
+    configured_speed: "Auto"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: ""
+    link_trap: ""
+    port: "1/23"
+  - admin_status: ""
+    auto_negotiation: "Enable"
+    configured_duplex: "Auto"
+    configured_fec: ""
+    configured_hybrid: "NA"
+    configured_pause: ""
+    configured_speed: "Auto"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: ""
+    link_trap: ""
+    port: "1/24"
+  - admin_status: ""
+    auto_negotiation: "Enable"
+    configured_duplex: "Full"
+    configured_fec: ""
+    configured_hybrid: "NA"
+    configured_pause: ""
+    configured_speed: "1000"
+    detect_hybrid: "NA"
+    detected_duplex: "Full"
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: "1000"
+    eee: ""
+    link_trap: ""
+    port: "1/25"
+  - admin_status: ""
+    auto_negotiation: "Enable"
+    configured_duplex: "Full"
+    configured_fec: ""
+    configured_hybrid: "NA"
+    configured_pause: ""
+    configured_speed: "1000"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
```

Comment:
```suggestion
    detected_duplex: "Full"
    detected_fec: ""
    detected_hybrid: "NA"
    detected_pause: ""
    detected_speed: "1000"
    eee: ""
    link_trap: ""
    port: "1/1"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/2"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/3"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/4"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/5"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/6"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/7"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/8"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/9"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/10"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/11"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/12"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/13"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/14"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/15"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/16"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/17"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/18"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/19"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/20"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/21"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/22"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/23"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Auto"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "Auto"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: ""
    link_trap: ""
    port: "1/24"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Full"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "1000"
    detected_duplex: "Full"
    detected_fec: ""
    detected_hybrid: "NA"
    detected_pause: ""
    detected_speed: "1000"
    eee: ""
    link_trap: ""
    port: "1/25"
  - admin_status: ""
    auto_negotiation: "Enable"
    configured_duplex: "Full"
    configured_fec: ""
    configured_hybrid: "NA"
    configured_pause: ""
    configured_speed: "1000"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
```

Updated parsed data for `DETECT_HYBRID` >> `DETECTED_HYBRID` change to match other capture group names.